src/store/ducks/jurisdictions.ts

```diff
@@ -96,6 +104,11 @@ export default function reducer(
         ...state,
         allJurisdictionIds: action.allJurisdictionIds,
```

Comment:
Looks like jurisdictions are still replaced instead of merged.  Is this the case?  Do we have a test for this?